hbase-client/src/main/java/org/apache/hadoop/hbase/client/RegionInfo.java

```diff
@@ -304,12 +304,11 @@ static String getShortNameToLog(RegionInfo...hris) {
   }
 
   /**
-   * @return Return a String of short, printable names for <code>hris</code>
-   * (usually encoded name) for us logging.
+   * @return Return a String of short, printable names for <code>hris</code> (usually encoded name)
+   *   for us logging.
    */
   static String getShortNameToLog(final List<RegionInfo> ris) {
-    return ris.stream().map(ri -> ri.getShortNameToLog()).
-    collect(Collectors.toList()).toString();
+    return ris.stream().map(ri -> ri.getShortNameToLog()).collect(Collectors.toList()).toString();
```

Comment:
If you like, `ri -> ri.getShortNameToLog()` can be replaced with method reference `RegionInfo::getShortNameToLog`. Same other places. Good to have class type before method so I personally prefer that.

Reply:
Yes. Thats better. Let me fix.